### PR TITLE
Update Scheduled Vulnerability Check workflow

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: .github/workflows/scheduled-vuln-check.yaml [hunk 1]
```diff
@@ -16,42 +16,42 @@ jobs:
       CR_PAT: ${{ secrets.CR_PAT }}
       SLACK_SECURITY_WEBHOOK_URL: ${{ secrets.SLACK_SECURITY_WEBHOOK_URL }}
 
-  call-vuln-check-for-v3_4:
```
Comment (brfrn169) [inline .github/workflows/scheduled-vuln-check.yaml:19]: 3.4 is EOL. So remove it.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**